test/topology_custom/test_raft_no_quorum.py

```diff
@@ -108,3 +108,44 @@ async def test_quorum_lost_during_node_join(manager: ManagerClient) -> None:
 
     logger.info("waiting for third node joining process to fail")
     await third_node_future
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_cannot_run_operations(manager: ManagerClient) -> None:
+    logger.info("starting a first node (the leader)")
+    servers = [await manager.server_add(config={
+        'error_injections_at_startup': [
+            {
+                'name': 'group0-raft-op-timeout-in-ms',
+                'value': '1000'
+            },
+            {
+                'name': 'raft-group-registry-fd-threshold-in-ms',
+                'value': '500'
+            }
+        ]
+    })]
+
+    logger.info("starting a second node (a follower)")
+    servers += [await manager.server_add()]
+
+    logger.info("stopping the second node")
+    await manager.server_stop_gracefully(servers[1].server_id)
+
+    logger.info("attempting removenode for the second node")
+    await manager.remove_node(servers[0].server_id, servers[1].server_id,
+                              expected_error="raft operation [read_barrier] timed out, no group0 quorum",
+                              timeout=60)
```

Comment:
I understand these will timeout much faster than 60s (due to the server-side error injections)?
In that case I wonder why you decided to add `timeout` parameter to these functions instead of still relying on `TOPOLOGY_TIMEOUT` (not saying it's a bad thing, just wondering)

Reply:
Initially, because of the injections, I used much smaller timeouts, 5 or 10 seconds. This allows you to waste less time in development. But then this started to fail on debug CI and I reverted everything back to the default 60s.

Now I'm starting to think we shouldn't have this timeout arguments at all. We should use one default timeout for everything, say, 5 seconds for 'normal' builds, 60 seconds for debug. Tests should ensure (through injections) that their operations are within that limit.

Reply:
> Initially, because of the injections, I used much smaller timeouts, 5 or 10 seconds. This allows you to waste less time in development. But then this started to fail on debug CI and I reverted everything back to the default 60s.

Wait but doesn't it timeout after 1 second right now? (because of the injections)

The 60s timeout is only for the client side to fail the test earlier **if** the expected server-side timeout doesn't happen, IIUC?

Reply:
> Wait but doesn't it timeout after 1 second right now? (because of the injections)

Yes, but who knows what happens on CI in debug mode. Actual delays here correlate quite loosely with the expected ones.

> The 60s timeout is only for the client side to fail the test earlier if the expected server-side timeout doesn't happen, IIUC?

Yes, but on CI in debug we should account for a lot of arbitrary delays. I hope 60s is enough for that.